src/Components/Footer/Footer.tsx

```diff
@@ -135,11 +131,6 @@ export const Footer: React.FC<FooterProps> = props => {
             </Text>
 
             <Text variant="sm">
-              {!enableNewSWALandingPage && (
-                <FooterLink my={2} to="/meet-the-specialists">
-                  Talk to a Specialist
-                </FooterLink>
-              )}
```

Comment:
The changes are not hidden behind a feature flag. I believe this aligns with the requirements
In addition, removed the ff from here ^^^